kyo-prelude/jvm/src/test/scala/kyo2/kernel/BytecodeTest.scala

```diff
@@ -37,7 +37,7 @@ class BytecodeTest extends Test:
 
     "handle" in {
         val map = methodBytecodeSize[TestHandle]
-        assert(map == Map("test" -> 50, "anonfun" -> 17, "handleLoop" -> 273))
+        assert(map == Map("test" -> 30, "anonfun" -> 15, "handleLoop" -> 266))
```

Comment:
Reduced size because `accept` in `Effect.handle` is now `inline`. I'm not sure why I didn't do this initially, it might be an oversight.